djangocms_versioning/admin.py

```diff
@@ -135,6 +155,194 @@ def has_change_permission(self, request, obj=None):
         return super().has_change_permission(request, obj)
 
 
+class ExtendedVersionAdminMixin(VersioningAdminMixin):
+    """
+    Extended VersionAdminMixin for common/generic versioning admin items
+    """
+    list_display_links = None
+    """
+    By default preview_implemented is False, when implemented in admin class that used mixin,
+    set to True if preview is implemented!
+    """
+    preview_implemented = False
+
+    class Media:
+        js = ("admin/js/jquery.init.js", "djangocms_versioning/js/actions.js")
+        # Verify whether version locking is installed before using its assets!
+        if using_version_lock:
+            css = {
+                "all": ("djangocms_versioning/css/actions.css", "djangocms_version_locking/css/version-locking.css",)
+            }
+        else:
+            css = {
+                "all": ("djangocms_versioning/css/actions.css",)
+            }
+
+    def get_version(self, obj):
+        """
+        Return the latest version of a given object
+        :param obj: Versioned Content instance
+        :return: Latest Version linked with content instance
+        """
+        return obj.versions.all()[0]
```

Comment:
I'm not sure why this is not just first. Could be a duplicate call if using .first(), probably better to leave it as it is for now to prevent any regression.